gui/web/src/components/molecules/Welcome/Welcome.js

```diff
@@ -61,15 +61,15 @@ class Welcome extends Component {
           </h3>
 
           <p className={styles.text}>
-          Kelp is a free and open-source trading bot for the Stellar Decentralized Exchange and centralized exchanges.
+            "Kelp is a free and open-source trading bot for the Stellar Decentralized Exchange and centralized exchanges. Kelp is programmed to support multiple trading schemes. This GUI of Kelp is limited to the Stellar Decentralized Exchange with only the buysell scheme.
           </p>
 
           <p className={styles.text}>
-          Kelp comes with multiple trading strategies out-of-the-box. This GUI of Kelp is limited to the Stellar Decentralized Exchange with the buysell strategy only.
+            You can use this GUI to define your own parameters to quickly get up and running with a trading bot in a matter of minutes.
```

Comment:
`for each trading scheme` needs to be added between `parameters` and `to quickly get up and running...`

not sure if this modification was made in the doc before or after your PR